docs/libertyDev.md

```diff
@@ -63,20 +63,3 @@ Tests can read the following system properties to obtain information about the L
 | liberty.http.port | The port used for client HTTP requests. |
 | liberty.https.port | The port used for client HTTP requests secured with SSL (HTTPS). |
 
-In order to properly propagate the system properties from the Gradle JVM running dev mode to the JVM(s) running your tests, you must configure your `build.gradle` to set the system properties for the test JVM(s).
```

Comment:
It might be a good idea to document what properties are being automatically propagated to the test JVM.

Reply:
@cherylking The `microshed` properties are duplicates of the `liberty.*` properties and are used by the MicroShed Testing framework.  They were not intended to be used directly by the user (even though they can be).    The user should use the liberty.* properties instead.
I agree that we can include an example of how to propagate additional properties though.